ethereum/contracts/relayer/deliveryProvider/DeliveryProviderGovernance.sol

```diff
@@ -326,4 +331,11 @@ abstract contract DeliveryProviderGovernance is
         }
         _;
     }
+
+    modifier onlyPricingWallet() {
+        if (pricingWallet() != _msgSender()) {
+            revert CallerMustBePricingWallet();
+        }
+        _;
+    }
```

Comment:
I'd actually make the price oracle role have a subset of functions allowed to the owner and administrators. I.e. the owner and the administrators should still be able to update the prices.

Reply:
done!